src/components/common/InputField.jsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import classNames from "classnames";
+import { Icon } from "react-icons/md";
```

Comment:
Icon never used? 
**Warning**
Line 3:10:  'Icon' is defined but never used            no-unused-vars